backend/course_evaluations/models.py

```diff
@@ -72,11 +71,22 @@ class CourseEvaluation(models.Model):
     # Many-to-many Relationship with the Django User model
     # related_name: Allows to reference `CourseEvaluation` from User model
     coordinators = models.ManyToManyField("auth.User", related_name="course_evaluation_coordinator")
-
     eoc_set = models.ForeignKey(EOCSet, on_delete=models.CASCADE)
-
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     def __str__(self):
         return f"{self.eoc_set.name} - {self.unit_code} ({self.created_at})"
+
+
+class CourseEvaluationJustification(models.Model):
+    """
+    A course coodinator must provide some written justification for how their course
+    is acheiving a variety of EOCs. It may be the case that a single written justification
+    can be used for multiple different EOC specifics.
+    """
+
+    course_evaluation = models.ForeignKey(CourseEvaluation, on_delete=models.CASCADE)
+    eoc_specific = models.ForeignKey(EOCSpecific, on_delete=models.CASCADE)
+    justification = models.TextField(null=False, blank=True)
```

Comment:
nice one! yeah this hould be the standard for any textfield... as it gets ambiguous sometimes if a value has `null` or `""`... in this case it is always `""` if empty

Reply:
👍